Rocket.Chat/Models/Subscription/SubscriptionCanInviteUsers.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  SubscriptionCanInviteUsers.swift
+//  Rocket.Chat
+//
+//  Created by Matheus Cardoso on 5/21/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+
+extension Subscription {
+    func canInviteUsers(user: User? = AuthManager.currentUser()) -> Bool {
+        guard let user = user else {
+            return false
+        }
+
+        if isJoined() && user.hasPermission(.addUserToJoinedRoom, subscription: self) {
+            return true
+        }
+
+        if type == .channel && user.hasPermission(.addUserToAnyChannelRoom, subscription: self) {
+            return true
+        }
+
+        if type == .group && user.hasPermission(.addUserToAnyPrivateRoom, subscription: self) {
+            return true
+        }
+
+        return false
```

Comment:
As an admin, as the creator of the room, I was not able to invite users to the room... I think the logic here is not quite right. On the web, I can do it. Can you take a look please?

Reply:
Are you sure @rafaelks ? Can you provide steps to reproduce this? The permissions are working as expected for me.

Maybe there's some delay with fetching them. I'll do some tests.

Reply:
@cardoso It worked for me now... I think was some kind of delay. 👍

Reply:
> As an admin, as the creator of the room, I was not able to invite users to the room... I think the logic here is not quite right. On the web, I can do it. Can you take a look, please?

HI I have the same problem with who create a channel or group cant invite another user 
I think who create a group that person become the admin for that room so he should have access to add another person @rafaelks how its work my native IOS does not work